packages/fakta-uttaksdokumentasjon/src/components/UttakDokumentasjonFaktaTable/UttakVurderingStatus.tsx

```diff
@@ -12,14 +12,15 @@ interface Props {
 }
 
 const UttakVurderingStatus: FC<Props> = ({ vurdering, morsStillingsprosent }) => {
+  if (!vurdering) {
```

Comment:
Isåfall burde Props typen også utvides til å ha `null` som en mulig verdi hvis det er det som trigget feilen:

```
interface Props {
  vurdering?: UttakVurdering | null;
  morsStillingsprosent?: number;
}
```

Reply:
Er ikke dette implisitt av vudering?:
Betyr ikke det at den kan være null eller undefined?